components/raftstore/src/store/fsm/store.rs

```diff
@@ -246,6 +247,21 @@ where
     }
 }
 
+#[derive(Default)]
+pub struct PeerTickBatch {
+    pub ticks: Vec<Box<dyn FnOnce() + Send>>,
+    pub wait_duration: Duration,
```

Comment:
Maybe we can remove it, for the same tick the timeout should be the same, we can get the timeout from config when flushing ticks.

Reply:
Ok